TrafficSimulator/Assets/Prefabs/Car/AutoDrive.cs

```diff
@@ -614,11 +613,10 @@ private void P_UpdateTargetAndCurrent()
             {
                 _currentNode = _target;
                 // When the currentNode is changed, the navigation path needs to be updated
-                if (ShowNavigationPath)
-                    Navigation.DrawPathRemoveOldestPoint(_navigationPathContainer);
-                _totalDistance += _currentNode.DistanceToPrevNode;
-                _target = GetNextLaneNode(_target, 0, _roadEndBehaviour == RoadEndBehaviour.Loop);
-                _lerpSpeed = _speed;
+                Navigation.DrawPathRemoveOldestPoint(_navigationPathContainer);
```

Comment:
Why was the if statement here removed?

Reply:
Yes but this function is not related to the navigation path updating, it just draws the path. The drawing of the path is unrelated to the path updating itself